src/components/header/renderer.tsx

```diff
@@ -157,14 +161,31 @@ class Header extends React.Component<Props, State> {
     this.setState({
       invalidUrl: !gistCommits.ok,
     });
+    if (filename.length === 0 && this.state.invalidUrl) {
+      this.setState({
+        invalidFilename: false,
+      });
+    }
+    if (revision.length === 0 && this.state.invalidUrl) {
```

Comment:
same condition as above

Reply:
This removes the error message of revision if it is empty and if gist-url is false. If this was removed something like this would occur:
![corrected11](https://user-images.githubusercontent.com/35253743/53829197-972d6880-3fa5-11e9-8544-2a1f06cad552.png)

Reply:
I still don't get why we can't just write

```
if (state.invalidURL) {
  this.setState({
    invalidFileName: filename.length !== 0,
	invalidRevision: revision.length !== 0
  })
}
```